src/Common/Core/Impl/Security/SecurityUtilities.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+namespace Microsoft.Common.Core.Security {
+    public static class SecurityUtilities {
+        public static IntPtr CreatePasswordBuffer() {
+            return CreateSecureStringBuffer(NativeMethods.CREDUI_MAX_PASSWORD_LENGTH);
+        }
+
+        public static SecureString ToSecureString(this string s) {
```

Comment:
The only place I need it is with `Find(Claims.Password)` which returns plain text string :-(